### PR TITLE
Remove deprecated each usage (#1424)

### LINKED ISSUE #1423: \Elastica\Transport\Http uses deprecated each() function
`\Elastica\Transport\Http` code uses `each()` function which is deprecated as of PHP 7.2. This results in flooding logs with `E_DEPRECATED` entries. 

https://github.com/ruflin/Elastica/blob/23f2bf0c7cfdc15a219ada33bb17a4367b5ffddb/lib/Elastica/Transport/Http.php#L118

### PR DESCRIPTION
**Pull request by franmomu**
* Change each to foreach in Http transport code
* Change each to current & key && next in MultiBuilder

Closes  #1423

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ All notable changes to this project will be documented in this file based on the
 ### Backward Compatibility Breaks
 
 ### Bugfixes
+- Characters "<" and ">" will be removed when a query term is passed to [`Util::escapeTerm`](https://github.com/ruflin/Elastica/pull/1415/files). Since v5.1 the [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/5.1/query-dsl-query-string-query.html#_reserved_characters) states that these symbols cannot be escaped ever.
+- Remove [`each()`](http://www.php.net/each) usage to fix PHP 7.2 compatibility
+
+### Added
+
+* Added request parameters to `Client->deleteDocuments()`. [#1419](https://github.com/ruflin/Elastica/pull/1419)
```
Comment (ruflin) [inline CHANGELOG.md]: I think a few more things sneaked into the changelog here?

Comment (franmomu) [reply CHANGELOG.md]: oops, sorry, I cherry-picked the commit and I didn't notice this 😬

**Comment by ruflin** [comment]
@franmomu Thanks for the fix.

**Comment by d-claassen** [comment]
Hi @ruflin, any chance for a 5.3.2 release including this fix? We just upgraded to PHP7.2 and our error log has exploded due to this. A new release would be really nice to have!

**Comment by d-claassen** [comment]
I missed the 5.x branch. We'll configure composer to depend on that until we upgrade to v6, thanks!

**Comment by ruflin** [comment]
@stof Here you go: https://github.com/ruflin/Elastica/releases/tag/5.3.2

**Comment by stof** [comment]
@ruflin thanks

**Pull request merged**